code-experiments/src/coco.h

```diff
@@ -488,6 +542,20 @@ double coco_random_normal(coco_random_state_t *state);
 
 /***********************************************************************************************************/
 
+/**
+ * @name Methods regarding noisy problems
+ */
+/**@{*/
+
+/**
+ * @brief Resets seeds
+ */
+void reset_seeds(void);
```

Comment:
I put it because I needed the seeds to be the same each time I ran a regression test over the new `bbob-noisy` suite against the data from the legacy code. I don't know if this is actually the wanted behavior. I guess those seeds will be automatically reset every time the library is built (the seeds are global variables, I'm not sure if this is the best approach, but this is how it was done in the legacy code, so I just decided to stick with it). So, when the regression tests are ran at each commit to the repository, I guess it should still be good as it builds it before (?)... I think that, if this is the case, this function wouldn't be necessary and we could delete it. I only defined it because I was running the tests locally. For now I'll just add the  `coco_` prefix as you said, I'll wait for you confirmation to drop it.